static/src/javascripts/projects/membership/contributions-recurring-tab.js

```diff
@@ -50,11 +63,41 @@ const hideCancelContribution = (): void => {
     $(CANCEL_CONTRIBUTION).addClass(IS_HIDDEN_CLASSNAME);
 };
 
-const hideContributionInfo = (): void => {
-    $(CONTRIBUTION_INFO).addClass(IS_HIDDEN_CLASSNAME);
+const displayCancelContributionForm = (): void => {
+    $(CANCEL_CONTRIBUTION_FORM).removeClass(IS_HIDDEN_CLASSNAME);
+};
+
+const hideCancelContributionForm = (): void => {
+    $(CANCEL_CONTRIBUTION_FORM).addClass(IS_HIDDEN_CLASSNAME);
+};
+
+const hideCancelContributionSuccessMessage = () => {
```

Comment:
Do you fancy adding type annotations to these functions? 😄

Reply:
My pleasure 👍